android/sdl_android/src/androidTest/java/com/smartdevicelink/protocol/SdlPacketTests.java

```diff
@@ -60,6 +64,30 @@ public void testNull() {
         assertEquals(TestValues.GENERAL_STRING, sdlPacket.getTransportRecord().getAddress());
     }
 
+    @Test
+    public void testVehicleMake() {
+        HashMap<String, Object> testMap = new HashMap<>();
+        testMap.put(VEHICLE_MAKE, "sdl");
+        testMap.put(VEHICLE_MODEL, "sdl");
+        testMap.put(VEHICLE_TRIM, "sdl");
+        testMap.put(VEHICLE_MODEL_YEAR, "2019");
+
+        byte[] testPayload = BsonEncoder.encodeToBytes(testMap);
+
+
+        SdlPacket sdlPacket = new SdlPacket();
+
+        assertNull(sdlPacket.getTag(VEHICLE_MAKE));
+        assertNull(sdlPacket.getTag(VEHICLE_MODEL));
+        assertNull(sdlPacket.getTag(VEHICLE_MODEL_YEAR));
```

Comment:
@kostyaBoss I would swap lines 82 and 83.  (So the order ends up being `VEHICLE_MAKE`, `VEHICLE_MODEL`, `VEHICLE_TRIM`, `VEHICLE_MODEL_YEAR`) like the other two sections. (70-73 for example)